sqs_common/sqs.go

```diff
@@ -97,6 +98,11 @@ func (s *SQS) DeleteMessages(m []*sqs.Message) error {
 	}
 
 	_, err := s.svc.DeleteMessageBatch(deleter)
+	if err != nil {
+		lg.Error("deleting messages failed, retrying", log.KVErr(err))
+		//try again, this is important
+		_, err = s.svc.DeleteMessageBatch(deleter)
```

Comment:
I would add a second error message here saying the retry failed, and warn that "message may be duplicated later"